tests/test_whistle.py

```
@@ -45,13 +97,74 @@
 }
 
 
+def _fake_query_rules(query_texts, n_results=5, where=None):
+    """Return filtered rule results based on where clause."""
+    docs, metas, dists = [], [], []
+    for d, m in zip(_FAKE_RULE_DOCS, _FAKE_RULE_METAS):
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)